test/unit/model/test_objective.py

```diff
@@ -20,7 +20,7 @@ def test_call(self, model_creator_mock, mea_mock, split_data_mock):
 
         objective = XGBRegressorObjective(input_data)
         trial_mock = MagicMock()
-        error = objective(trial_mock)
+        objective(trial_mock)
 
```

Comment:
Is this part of the Tracy PR or the hyperparameters PR?

Reply:
No not really. This was a small/bug leftover which is already solved upstream.